src/content/operations/usage/_index.en.md

```diff
@@ -981,3 +981,75 @@ Switched to context "cluster3".
 $ kubectl delete namespace nginx-test
 namespace "nginx-test" deleted
 ```
+
+### Migrating existing applications to use MultiCluster Services
+
+To use MultiCluster Services applications need to use `clusterset.local` suffix. There are two main ways to achieve this
+without needing to rewrite the applications.
+
+#### Use DnsConfig
+
+First option is to use [dnsConfig][dnsconfig doc] field in the `Deployment` or `Pod` spec of the applications to add `clusterset.local`
+to the search path in applications' `/etc/hosts/` file.
+
+```bash
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: default
+  name: dns-example
+spec:
+  containers:
+    - name: test
+      image: nginx
+  dnsPolicy: "None"
+  dnsConfig:
+    nameservers:
+      - 192.0.2.1 # this is an example
+    searches:
+      - default.svc.cluserset.local
+      - ns1.svc.cluserset.local # example namespace
+      - svc.clusterset.local
+      - clusterset.local
+```
+
+##### Pros of DnsConfig
+
+* Useful when specific applications want to use multiple MultiCluster Services.
+* Preferred if some applications want to continue using local version of the Service
+
+##### Cons of DnsConfig
+
+*Need to modify `Deployment` or `Pod` spec of all applications that need to use MultiCluster Service.
+
+#### Use ExternalName Service
+
+Second option is to create a Service of type `ExternalName` that points to MultiClusterService.
+
+```bash
+apiVersion: v1
+kind: Service
+metadata:
+  name: mcs-sample
+  namespace: demo
+spec:
+  type: ExternalName
+  sessionAffinity: None
+  externalName: mcs-sample.demo.svc.clusterset.local
+```
+
+This will make DNS replace any queries to local service `mcs-sample.demo.cluster.local` with the MultiCluster Service
+`mcs-sample.demo.svc.clusterset.local`. This can be also used to redirect to a MultiCluster Service in a different
+namespace.
```

Comment:
```suggestion
This will make DNS replace any queries to the local `mcs-sample.demo.cluster.local` service with the MultiCluster
`mcs-sample.demo.svc.clusterset.local` service.
This can be also used to redirect to a MultiCluster Service in a different namespace.
```